apps/api/src/server/appeals/appeals/appeals.validators.js

```diff
@@ -1,10 +1,51 @@
 import { composeMiddleware } from '@pins/express';
-import { param } from 'express-validator';
+import { param, query } from 'express-validator';
 import { validationErrorHandler } from '../../middleware/error-handler.js';
 
+/**
+ * @param {string} value
+ * @returns {string}
+ */
+const hasValue = (value) => value;
```

Comment:
It'd be clearer if this returned a boolean, using e.g. `=> Boolean(value)`, since based on its usage we don't need the string return value.